Samples/BackgroundBlur/BackgroundBlur/common/PropVar.h

```diff
@@ -0,0 +1,95 @@
+#pragma once
+
+#include <propsys.h>
+#include <propvarutil.h>
+
+///////////////////////////////////////////////////////////////////////////////
+//
+// PropVariant Class
+//
+// Wrapper class for PROPVARIANT types. 
+//
+// Notes: 
+//    This class offers accessor functions for a subset of the PROPVARIANT
+//    data types, designed to handle all of the types in actual use by 
+//    Media Foundation at this time.
+// 
+///////////////////////////////////////////////////////////////////////////////
+
+class PropVariant : public PROPVARIANT
```

Comment:
There's std::variant, not sure if it covers your use case

Reply:
Unused code from MF samples, I'll remove from the project.